### DISCUSSION TITLE
Control Global Midi Channel

### DISCUSSION THREAD

**Discussion by phillix2** [shanteacontrols/OpenDeck#230] [Q&A]
Is it possible to have a potentiometer controlling the Global Midi Channel? I would easily been able to "multiply" the number of different messages I could send with the same potentiometers and buttons, as my soft synth can map the CC midi channel to different parameters.

Thanks.

**Comment by paradajz** [comment]
That setting isn't mappable.

**Reply by phillix2** [reply]
Oks. Tks for answering.
